### PR TITLE
reshuffled database things into DataAccess class

### PR DESCRIPTION
**Pull request by helgridly**
This makes it slightly harder to get access to the table components by doing `dbRef.some_db_query`, which is almost certainly not what you want. (You want to call `dbRef.inTransaction` and get a `DataAccess` from there instead.)

- [ ] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Make sure Swagger is updated if API changes
- [ ] **Submitter**: If updating admin endpoints, also update [firecloud-admin-cli](https://github.com/broadinstitute/firecloud-admin-cli)
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [ ] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [ ] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [ ] Tell the tech lead (TL) that the PR exists if they wants to look at it
- [ ] Anoint a lead reviewer (LR). **Assign PR to LR**
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Product Owner** sign off
- [ ] **Assign to submitter** to finalize
- [ ] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Squash commits and merge to develop
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Verify swagger UI on dev environment still works after deployment
- [ ] **Submitter**: Inform other teams of any API changes via Slack and/or email
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@ before_install:
   - docker rm -f mysql || true          # in case the after_success step does not run
   - docker pull mysql/mysql-server:5.6
   # port 3306 is not available because Travis runs its own MySQL after 2006-12-01
-  - docker run -p 3307:3306 --name mysql -e MYSQL_ROOT_PASSWORD=leonardo-test -e MYSQL_USER=leonardo-test -e MYSQL_PASSWORD=leonardo-test -e MYSQL_DATABASE=testdb -d mysql/mysql-server:5.6
+  - docker run -p 3307:3306 --name mysql -e MYSQL_ROOT_PASSWORD=leonardo-test -e MYSQL_USER=leonardo-test -e MYSQL_PASSWORD=leonardo-test -e MYSQL_DATABASE=leotestdb -d mysql/mysql-server:5.6
```
Comment (helgridly) [inline .travis.yml]: I changed this because I run a local mysql and this is the same db name as Rawls'.

**Comment by helgridly** [comment]
This passes tests for me but my IntelliJ _hates_ our `*RouteSpec` classes now :/

File: .travis.yml (hunk 1 above)
Comment (rtitle) [reply .travis.yml]: Looks like this is for Travis? Do you want to change `docker/run-mysql.sh` too?

Comment (helgridly) [reply .travis.yml]: oh, good catch

**Comment by jmthibault79** [comment]
I don't see how this change solves the stated problem.  What's stopping us from writing `dbRef.dataAccess.some_db_query` ?

**Pull request merged**